src/data/emacs-mode/agda-input.el

```diff
@@ -233,6 +235,8 @@ order for the change to take effect."
   ("squb="  . ("⊑"))  ("squp="  . ("⊒"))
   ("squb=n" . ("⋢"))  ("squp=n" . ("⋣"))
 
+  ("<"    . ("＜")) (">"    . ("＞"))
```

Comment:
Perhaps these could be part of `\leq`/`\geq`.

Reply:
Hmm, I suppose that would be consistent since `\leq`/`\geq` already contain other characters that don't have any `eq` to them.  Although that does seem weird to me.  I would not have guessed `\leq` as a place to find ≪ or ≺.  Wouldn't it make more sense for those to also be in `\<`?

Reply:
I think the idea is to include "everything" in the bindings `\leq`, `\leqn`, `\geq` and `\geqn`, but to also have dedicated bindings for certain characters:

https://github.com/agda/agda/blob/4e6428395398907a722e0563d2813d7eaeb32248/src/data/emacs-mode/agda-input.el#L212-L234

Reply:
Ah, I see.  Would it make sense to *also* have ≪, ≺, and so on under `\<`?  I don't find it intuitive to have to look under `\leq` to get a character that doesn't have a `q` in it.

By the same principle, I guess the fullwidth equals sign should also be under `\eq` then.

Reply:
> Would it make sense to _also_ have ≪, ≺, and so on under `\<`?

Yes, I guess that makes sense.

> By the same principle, I guess the fullwidth equals sign should also be under `\eq` then.

Yes.

Reply:
Ok, I put the fullwidth equals sign in `\eq`.  Should I put ≪, ≺, etc. into `\<` in this PR, or make a new one since that's not related to fullwidth characters?

Reply:
I suggest that you add `\<` in a separate commit in the same pull request.

Reply:
Ok, done!

Reply:
Thanks. Please rebase to avoid having too much noise in the commit history. (I think two commits suffice for these changes: one that adds `\<`/`\>` with the existing characters, and one that adds the fullwidth characters.)